Plugins/ScriptPlugins/ParserPlutoniumT5COZM.js

```diff
@@ -0,0 +1,49 @@
+var rconParser;
+var eventParser;
+
+var plugin = {
+    author: 'RaidMax, SwordSWD',
+    version: 0.1,
+    name: 'Plutonium T5 CO-OP/Zombies Parser',
+    isParser: true,
+
+    onEventAsync: function (gameEvent, server) {
+    },
+
+    onLoadAsync: function (manager) {
+        rconParser = manager.GenerateDynamicRConParser(this.name);
+        eventParser = manager.GenerateDynamicEventParser(this.name);
+
+        rconParser.Configuration.DefaultInstallationDirectoryHint = '{LocalAppData}/Plutonium/storage/t5';
+        rconParser.Configuration.CommandPrefixes.RConResponse = '\xff\xff\xff\xffprint\n';
+        rconParser.Configuration.Dvar.Pattern = '^(?:\\^7)?\\"(.+)\\" is: \\"(.+)?\\" default: \\"(.+)?\\"\\n?(?:latched: \\"(.+)?\\"\\n)?\\w*(.+)*$';
+        rconParser.Configuration.CommandPrefixes.Tell = 'tell {0} {1}';
+        rconParser.Configuration.CommandPrefixes.RConGetInfo = undefined;
+        rconParser.Configuration.GuidNumberStyle = 7; // Integer
+        rconParser.Configuration.DefaultRConPort = 3074;
+        rconParser.Configuration.CanGenerateLogPath = false;
+        rconParser.Configuration.CommandPrefixes.Kick         = 'clientkick {0}';
```

Comment:
Replace `'clientkick {0}'` with `'clientkick {0} "{1}"'`, it works

Reply:
fixed